build.sbt

```diff
@@ -168,7 +171,12 @@ lazy val root = project.in(file(".")).
 lazy val core = crossProject.in(file("core")).
   settings(commonSettings: _*).
   settings(
-    name := "fs2-core"
+    name := "fs2-core",
+    resolvers += Resolver.bintrayRepo("djspiewak", "maven"), // Temporary until cats-effect is published on Maven Central
+    libraryDependencies ++= Seq(
+      "org.typelevel" %%% "cats-core" % "0.9.0",
+      "cats-effect" %%% "cats-effect" % "0.1-650a34e"
```

Comment:
cats-effect is now [published](http://search.maven.org/#search%7Cga%7C1%7Cg%3A%22org.typelevel%22%20AND%20a%3Acats-effect*) on Maven Central.

```sbt
"org.typelevel" %%% "cats-effect" % "0.1-d1b5231"
```